docs/docs/spark-configuration.md

```diff
@@ -165,6 +165,8 @@ spark.read
 | vectorization-enabled  | As per table property | Overrides this table's read.parquet.vectorization.enabled                                          |
 | batch-size  | As per table property | Overrides this table's read.parquet.vectorization.batch-size                                          |
 | stream-from-timestamp | (none) | A timestamp in milliseconds to stream from; if before the oldest known ancestor snapshot, the oldest will be used |
+| streaming-max-files-per-micro-batch | INT_MAX | Maximum number of files per microbatch |
+| streaming-max-rows-per-micro-batch  | INT_MAX | Maximum number of rows per microbatch. This number should be greater than the number of records in any data file in the table. The smallest unit that will be streamed is a single file, so if a data file contains more records than this limit, the stream will get stuck at this file.|
```

Comment:
nit: we try to keep these  descriptions small. I think you can add a * or note if you have more info you want in here, especially since it seems like we are just describing a  bug

Reply:
Ack ! 
Here is what it looks like now, please let me know your thoughts 
<img width="733" alt="Screenshot 2025-02-19 at 10 51 59 AM" src="https://github.com/user-attachments/assets/4ce98100-6d1f-453a-b729-0b404935bbbb" />

Reply:
Yes, it makes sense to add the caveat as a Note or Warning.